app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
+  after_save :update_post_counter
+
+  def update_post_counter
+    author.increment!(:posts_counter)
+  end
+
+  def recents_comments
+    comments.order(created_at: :desc).limit(5)
+  end
```

Comment:
- [ ] Great job creating a method that updates the posts counter for a user. 👏🏻 👏🏻  However It would be nice to make the `update_post_count` method to be a `private` method 👍🏻 💯 Kindly make sure to add this to the remaining model as well ✍🏻 

   ```ruby
     # app/models/post.rb
     after_save :update_post_counter
   
     private
   
     def update_post_counter
       author.increment!(:posts_counter)
     end
   ```

- This is also valid on the `# app/models/comment.rb` and `# app/models/like.rb` model's methods must be a private